pkg/anchor/builder/context.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package builder
+
+const (
+	// this context is pre-loaded by aries framework.
+	defVCContextURI = "https://www.w3.org/2018/credentials/v1"
+
+	// AnchorContextURI is anchor credential context URI.
+	AnchorContextURI = "https://trustbloc.github.io/did-method-orb/contexts/anchor/v1"
+
+	// JwsContextURI is jws context.
+	JwsContextURI = "https://w3id.org/jws/v1"
+)
+
+// AnchorContext is anchor context.
+const AnchorContext = `
```

Comment:
Note we could end up with multiple versions over time.

Reply:
fixed